scylla/src/transport/topology.rs

```diff
@@ -70,7 +70,7 @@ impl TopologyReader {
         TopologyReader {
             control_connections,
             connect_port,
-            connection_config
+            connection_config,
```

Comment:
Nit: Please move the comma fixes to the previous commit. Ideally, every commit should pass linter checks.

Reply:
Fixed, sorry I thought I checked it. I have to make some git hooks to prevent this.